tests/test_calibration.py

```diff
@@ -235,3 +237,8 @@ def test_AGIPD_CalibrationData_report():
     assert set(agipd_cd) == {"Offset", "Noise", "ThresholdsDark", "BadPixelsDark"}
     assert agipd_cd.aggregator_names == [f"AGIPD{n:02}" for n in range(16)]
     assert isinstance(agipd_cd["Offset", "AGIPD00"], SingleConstant)
+
+
+def test_format_time(mock_spb_aux_run):
+    assert CalCatAPIClient.format_time(mock_spb_aux_run).startswith(
+        CalCatAPIClient.format_time(datetime.now())[:8])
```

Comment:
Do we also want to check that the formatted timestamp includes a timezone? I think checking that it contains one of `+zZ` ought to cover all possibilities.

Reply:
Good point, I refactored the test to actually use proper `datetime` objects. I figured relying on `format_time()` to return some ISO format is not a worse assumption than comparing strings...